docs/admin/install/knative-with-operators.md

```diff
@@ -197,139 +197,185 @@ NAME              VERSION             READY   REASON
 knative-serving   <version number>    True
 ```
 
-### Installing with different networking layers
+### Installing Knative Serving with different networking layers
 
-??? "Installing the Knative Serving component with different network layers"
+Knative Operator can configure Knative Serving component with different network layer options. Istio is the default network
+layer, if the ingress is not specified in the Knative Serving CR. Click on each of the following tabs to see how you can configure
+Knative Serving with different ingresses:
 
-    Knative Operator can configure Knative Serving component with different network layer options. Istio is the default network
-    layer, if the ingress is not specified in the Knative Serving CR. Click on each of the following tabs to see how you can configure
-    Knative Serving with different ingresses:
+=== "Istio (default)"
 
-    === "Ambassador"
+    The following commands install Istio and enable its Knative integration.
 
-        The following commands install Ambassador and enable its Knative integration.
+    1. [Install Istio](installing-istio.md).
 
-        1. Create a namespace to install Ambassador in:
-          ```bash
-          kubectl create namespace ambassador
-          ```
+    1. To configure Knative Serving to use Istio, copy the following YAML into a file:
+        ```yaml
+        apiVersion: operator.knative.dev/v1alpha1
+        kind: KnativeServing
+        metadata:
+          name: knative-serving
+          namespace: knative-serving
+        ```
 
-        1. Install Ambassador:
-          ```bash
-          kubectl apply --namespace ambassador \
-            --filename https://getambassador.io/yaml/ambassador/ambassador-crds.yaml \
-            --filename https://getambassador.io/yaml/ambassador/ambassador-rbac.yaml \
-            --filename https://getambassador.io/yaml/ambassador/ambassador-service.yaml
-          ```
+    1. If Istio is installed under a namespace other than the default `istio-system`, add the `spec.config.istio` field
+       to the YAML file as follows:
+        ```yaml
+        apiVersion: operator.knative.dev/v1alpha1
+        kind: KnativeServing
+        metadata:
+          name: knative-serving
+          namespace: knative-serving
+        spec:
+          config:
+            istio:
+              local-gateway.<local-gateway-namespace>.knative-local-gateway: "knative-local-gateway.<istio-namespace>.svc.cluster.local"
+        ```
 
-        1. Give Ambassador the required permissions:
-          ```bash
-          kubectl patch clusterrolebinding ambassador -p '{"subjects":[{"kind": "ServiceAccount", "name": "ambassador", "namespace": "ambassador"}]}'
-          ```
+        Where:
+        - `<local-gateway-namespace>` is the local gateway namespace, which is the same as Knative Serving namespace `knative-serving`.
+        - `<istio-namespace>` is the namespace where Istio is installed.
```

Comment:
```suggestion
        Where:
        
        - `<local-gateway-namespace>` is the local gateway namespace, which is the same as Knative Serving namespace `knative-serving`.
        - `<istio-namespace>` is the namespace where Istio is installed.
```

Reply:
The formatting is broken here (see screenshot below). I've added an extra line before the bullet points to format it correctly.

<img width="827" alt="Screen Shot 2021-10-06 at 18 57 44" src="https://user-images.githubusercontent.com/33425661/136257918-937773bd-697d-49e8-9dad-17031135b7ba.png">